uvod-do-programovani-2/slicing-metody-moduly/excs/zavody.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Stříbrná medaile
+demand: 4
+---
+
+Stříbrná medaile je sice úžasný úspěch, ale kdo by nechtěl vyhrát? Podívejme se, kolik chybělo stříbrnému běžci k vítězství.
+
+* Nejprve si vytvoř dvě proměnné, do kterých ulož čas vítěze a čas závodníka se stříbrnou medailí. Oba časy převeď na minuty a ulož jako číslo.
```

Comment:
Tady se předpokládá něco jako `hodiny * 60 + minuty + vteriny / 60`, takže desetinné číslo?

Reply:
Jo, už vidím, že v druhé úloze je to specifikováno